NDTensors/ext/examples/NDTensorCUDA.jl

```diff
@@ -0,0 +1,91 @@
+using CUDA
```

Comment:
Let's run these package extension examples from the tests. I've found that running examples in the test suite (even if you don't explicitly test anything, which still checks for errors) can help make sure the examples stay up to date.

We can use `include(joinpath(pkgdir(NDTensors), "ext", "examples", "NDTensorCUDA.jl"))`, for example, as a way to get the correct file location in a robust way.